benchmarks/benchmarks-schema.yaml

```diff
@@ -0,0 +1,28 @@
+id: https://github.com/CMSgov/qpp-measures-data/versions/0.0.1/benchmarks-schema.yaml
+$schema: http://json-schema.org/schema#
+type: array
+items: { $ref: #/definitions/benchmark }
+
+definitions:
+  benchmark:
+    type: object
+    properties:
+      measureId:
+        description: Corresponds to measure object's measureId.
+        type: string
+      performanceYear:
+        description: The year for which data is being submitted
+        type: string
+      benchmarkYear:
+        description: The year against which the data is be compared.
+        type: string
+      deciles:
+        description: List of 9 numbers representing the inclusive lower bound of Deciles 2 through 10. The top and bottom are implied to be 100 and 0 respectively for direct measures and 0 and 100 for inverse measures. The range of any given decile begins at it's lower bound and continues up to but does not include the subsequent decile's lower bound. If the subsequent decile's lower bound is equal to the current decile's lower bound, then that decile is undefined or, in other words, empty.
```

Comment:
Suggested revised version:
```
The deciles list object is a list of 9 numbers representing deciles of measurement values for a given measure, performance year and submission method. The 9 numbers represent the inclusive lower bounds of deciles 2 through 10. The upper and lower bounds of the measurement value range are implied to be 100 and 0 respectively for direct measures and 0 and 100 respectively for inverse measures. The range of any given decile begins at its lower bound and continues up to but does not include the subsequent decile's lower bound. If the subsequent decile's lower bound is equal to the current decile's lower bound, then that decile is undefined or, in other words, empty.
```

Changes:
* uncapitalized Deciles in the first sentence
* Added summary sentence at the beginning
* Replace top and bottom with upper and lower bounds
* Replace "it's lower bound" with "its lower bound"